### DISCUSSION TITLE
An exception has been thrown during the rendering of a template

### DISCUSSION THREAD

**Discussion by akmaladisa** [wintercms/winter#953] [Q&A] [answered]
I need help, how the folder directory structure should be when i upload my winter project to cPanel? because i get this error. TYSM
![image](https://github.com/wintercms/winter/assets/69964483/b1b6dd56-a915-4736-bca3-746278109942)

**Comment by WebVPF** [comment]
Sites are in the `public_html` folder:

📂 public_html
 ┣ 📂 site1.com
 ┣ 📂 site2.com
 ┣ 📂 site3.com
 ┗ ...

And already inside these folders are the files of your sites (projects):

📂 site1.com
 ┣ 📂 bootstrap
 ┣ 📂 config
 ┣ 📂 modules
 ┣ 📂 plugins
 ┣ 📂 storage
 ┣ 📂 tests
 ┣ 📂 themes
 ┣ 📂 vendor
 ┣ 📜 artisan
 ┣ 📜 composer.json
 ┣ 📜 composer.lock
 ┣ 📜 index.php
 ┣ 📜 phpunit.xml
 ┗ 📜 server.php

**Reply by akmaladisa** [reply]
Thank you so much for ur help, but i get 404 error instead.
![image](https://github.com/wintercms/winter/assets/69964483/8cda6c02-37e4-47f0-8d83-9928ed01ca13)

**Reply by WebVPF** [reply]
@akmaladisa Do you have an `akmaldi.my.id` domain?

**Reply by akmaladisa** [reply]
sure, i do
![image](https://github.com/wintercms/winter/assets/69964483/d3c040ce-95c2-446e-8cda-3e806ca697d6)

**Comment by LukeTowers** [comment, marked as answer]
I'm pretty sure this is because you uploaded your local cache files to your server. It looks like you're developing the site on Window and deployed it to a linux box. You should just be able to run `artisan cache:clear` or just delete all your cache files under storage/framework/cache

**Reply by WebVPF** [reply]
**How to clear cache in cPanel**

In cPanel, go to the Terminal section. Then run the commands:

```bash
cd ~/public_html/akmaldi.my.id
```

```bash
php artisan cache:clear
```

Commands can be copied and pasted into the terminal using the mouse through the context menu.
